### DISCUSSION TITLE
CitraVR (Beta) v0.3.2

### DISCUSSION THREAD

**Discussion by amwatson** [amwatson/CitraVR#41] [Announcements]
## What's Changed
NOTICE: this release will reset your settings. This is being done to unstick users who were having issues due to merged configs. I will have to do this one more time in the next 9 days -- sorry!
* Added "Lubos Immersive Mode"under "Settings > VR"
     *  Warning: Immersive Mode causes visual artifacts on some content, including 2D menus.  
     * Warning: Immersive Mode will require shader recompilation, meaning titles will stutter on the first run.
* Added DPAD on thumbrest: DPAD behavior on CitraVR now matches [praydog's UEVR](https://github.com/praydog/UEVR).  Which is convenient, because I can borrow their diagram to show you how it works:
![uevr](https://github.com/amwatson/CitraVR/assets/1646398/7fc6aa9d-5b99-4e3b-981e-4c2b52401703) 
[image credit](https://docs.google.com/spreadsheets/d/1ZcjCQwzPOltaRZnpYU5_HPihEDareZq_0Ww1DZQ4USw/edit#gid=1224938259)
*  Set default CPU level to 3 (reduces power draw)
* Fixed issue where users reported CitraVR rendering "half-width" 
* Lubos VR180 immersive mode by @amwatson in https://github.com/amwatson/CitraVR/pull/32
* Additional "Immersive Mode" that has a higher resolution by @DrBeef in https://github.com/amwatson/CitraVR/pull/33

## New Contributors
* @DrBeef made their first contribution in https://github.com/amwatson/CitraVR/pull/33

**Full Changelog**: https://github.com/amwatson/CitraVR/compare/v0.3.1...v0.3.2

<hr /><em>This discussion was created from the release <a href='https://github.com/amwatson/CitraVR/releases/tag/v0.3.2'>CitraVR (Beta) v0.3.2</a>.</em>

**Comment by pantoufle2298** [comment]
Amazing work, thank you so much!!
